tasks/installation.yml

```diff
@@ -10,6 +20,22 @@
   retries: 10
   delay: 2
 
+- name: check if icingadb table and user in mysql exist
+  shell: >
+    set -o pipefail;
```

Comment:
`ansible.builtin.shell` will default to `/bin/sh`, which can only guarantee POSIX-compliant features. `set -o pipefail` is a Bash-specific feature, and this might fail if run on a system where `/bin/sh` is not provided by Bash (most prominently, Debian-based systems).

If you want to keep using this, the module invocation will additionally need
```yaml
  …
  args:
    executable: /bin/bash
  …
```

See also the [examples in the module documentation](https://docs.ansible.com/ansible/latest/collections/ansible/builtin/shell_module.html#examples).